models/recurly__account_overview.sql

```diff
@@ -30,12 +30,12 @@ account_next_invoice as (
 final as ( 
 
     select 
-        account_history.account_id,
-        account_history.created_at as account_created_at,
-        account_history.email as account_email,
-        account_history.first_name as account_first_name,
-        account_history.last_name as account_last_name,
-        account_history.state as account_state,
+        account_history.account_id, 
+        account_history.created_at as account_created_at, 
+        account_history.email as account_email, 
+        account_history.first_name as account_first_name, 
+        account_history.last_name as account_last_name, 
+        account_history.state as account_state, 
```

Comment:
Mostly to prevent bringing in too many columns to start.

I don't think I brought in the following fields into the source model, should I add them in, then include downstream in the transforms?

- `account_city`
- `account_phone`
- `account_postal_code`
- `account_region`
- `account_street_1`
- `account_street_2`

Reply:
keep city/postal code/region